classes/cliTool/CommandLineTool.php

```diff
@@ -67,7 +67,11 @@ public function __construct($argv = [])
         $request->setRouter($router);
 
         // Initialize the locale and load generic plugins.
-        PluginRegistry::loadCategory('generic');
+        $contextDao = Application::getContextDAO(); /** @var ContextDAO $contextDao */
+        $contextFactory = $contextDao->getAll(); /** @var DAOResultFactory $contextFactory */
```

Comment:
Ah, if you want to make the code more generic, the `DAOResultFactory` has a `toIterator()` which you could use with a `foreach`.